src/refactor_nrepl/ns/prune_dependencies.clj

```diff
@@ -60,7 +61,9 @@
                       symbols-in-file)
                 (some (partial libspec-in-use-without-refer-all? libspec)
                       symbols-in-file))
-              (libspec-in-use-with-rename? libspec symbols-in-file))
+              (libspec-in-use-with-rename? libspec symbols-in-file)
+              (and (not libspec-as)
```

Comment:
The example in [clj-kondo-linters](https://github.com/clj-kondo/clj-kondo/blob/master/doc/linters.md#unused-namespace) use `(ns foo (:require [foo.specs]))`.

In other words, this will allow both `foo.specs` and `[foo.specs]` to remain, but isn't the wrapping what's used to signal that it's side-effecting in the clj-kondo convention?

Reply:
It doesn't seem like clj-kondo cares about brackets 
![image](https://user-images.githubusercontent.com/47952597/190853266-b2251dff-7845-4f04-887f-8db2f69480e2.png)